packages/desktop-client/src/components/reports/util.ts

```diff
@@ -13,23 +17,24 @@ export async function runAll(queries, cb) {
   cb(data);
 }
 
-export function index(data, field, mapper) {
-  const result = {};
+export function index<T>(data: T[], field: string, mapper?: (input) => string) {
```

Comment:
Was a pretty fun problem to solve. Here you go: https://gist.github.com/MatissJanis/307e49bbeec34f2975709e0ec3bc3100 :)

Reply:
Ah, I see. Thanks for this!